CHANGELOG.md

```diff
@@ -3,6 +3,139 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.0.0-alpha.13
+
+_Dec 23, 2022_
+
+We'd like to offer a big thanks to the 10 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Improved column menu design and API
+- 🌍 Improve Korean and Russian locales
+- 📚 Documentation improvements
+- 🐞 Bugfixes
+
+### `@mui/x-data-grid@6.0.0-alpha.13` / `@mui/x-data-grid-pro@6.0.0-alpha.13` / `@mui/x-data-grid-premium@6.0.0-alpha.13`
+
+#### Breaking changes
+
+From https://github.com/mui/mui-x/pull/6619
+
+- `column` and `currentColumn` recieved by `<GridColumnMenu />` and column menu items was renamed to `colDef`.
+- `DATA_GRID_DEFAULT_SLOTS_COMPONENTS` export has been removed.
+
+- Some of the components have been renamed for consistency.
+
+  **Community Package:**
+
+  ```diff
+  -<GridFilterMenuItem />
+  +<GridColumnMenuFilterItem />
+  ```
+
+  ```diff
+  -<HideGridColMenuItem />
+  +<GridColumnMenuHideItem />
+  ```
+
+  ```diff
+  -<GridColumnsMenuItem />
+  +<GridColumnMenuColumnsItem />
+  ```
+
+  ```diff
+  -<SortGridMenuItems />
+  +<GridColumnMenuSortItem />
+  ```
+
+  ```diff
+  -interface GridFilterItemProps
+  +interface GridColumnMenuItemProps
+  ```
+
+  **Pro package:**
+  ```diff
+  -<GridColumnPinningMenuItems />
+  +<GridColumnMenuPinningItem />
+  ```
+
+  **Premium package:**
+  ```diff
+  -<GridAggregationColumnMenuItem />
+  +<GridColumnMenuAggregationItem />
+  ```
+
+  ```diff
+  -<GridRowGroupingColumnMenuItems />
+  -<GridRowGroupableColumnMenuItems />
+  +<GridColumnMenuGroupingItem />
+  ```
+
+From https://github.com/mui/mui-x/pull/7224
```

Comment:
I usually used to remove these `From *` and just listed down all the breaking changes.

But in this specific use case, as we have breaking changes from multiple PRs, probably something like this may make more sense?

```
PR 1 Name and Details (#123) @Author

- Breaking change 1
- Breaking change 2
    - Sub item for breaking change 2
- Breaking change 3

PR 2 Name and Details (#456) @Author

- Breaking change 1
- Breaking change 2
- Breaking change 3
  - Sub item for breaking change 3
```

Reply:
Alright. thanks, @MBilalShafi, and @m4theushw for clearing it up. I've updated it so if you can take one last look that would be great.